### PR TITLE
Changes to execute Run in Unix

### PR DESCRIPTION
**Pull request by maririos**
- Rename of file Run.runtime.config, to run.runtime.config

- Changes in msbuild.sh

- Fix copy paste information in the config.json file and clean it to be specific for Build Tools

cc: @joperezr @Priya91 @wtgodbe @sokket @tyoverby

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -19,29 +13,17 @@
       "defaultValue": true
     },
     "Platform": {
-      "description": "Enables/Disables the restoration of packages.",
+      "description": "Sets the value of the platform.",
       "valueType": "property",
       "values": ["AnyCPU", "x86", "arm", "x64", "amd64"],
       "defaultValue": "x64"
```
Comment (Priya91) [inline config.json]: The defaultValue here should be AnyCPU ? Since we are building only managed..

**Comment by Priya91** [comment]
LGTM

File: config.json (hunk 1 above)
Comment (maririos) [inline config.json]: Changed! Thanks

**Comment by joperezr** [comment]
LGTM

**Pull request merged**